src/TestFramework/Microsoft.Rest.ClientRuntime.Azure.TestFramework/ExtMethods.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+namespace Microsoft.Rest.ClientRuntime.Azure.TestFramework
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Reflection;
+
+    public static class ExtMethods
```

Comment:
This is currently a generic class to park all extension methods. This particular class has extension methods for other than the dictionary and used beyond ConnectionStringclass. If we ever have more extension methods, would be the right time to refactor into individual extension method classes.

---
In reply to: [78655067](https://github.com/Azure/azure-sdk-for-net/pull/2328#discussion_r78655067) [](ancestors = 78655067)